simple-acquaintances/data/datagen.py

```diff
@@ -0,0 +1,386 @@
+"""
+Generates data for the simple-acquaintances PSL example.
+
+Use -h/--help for usage options.
+"""
+
+import argparse
```

Comment:
The imports needs to be grouped into:
 - standard library
 - third party
 - this project (of which there are none right now)